### PR TITLE
`lang/c++`: Add simple HTTP program

### PR DESCRIPTION
**Pull request by razvand**
Add source code, Makefile, prebuilt ELF, pre-etracted dynamic libraries. Document steps to (re)build and run in `README.md`.

### CODE REVIEW

File: lang/c++/http_server/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Simple C++ HTP Server
```
Comment (StefanJum) [inline lang/c++/http_server/README.md:1]:
```suggestion
# Simple C++ HTTP Server
```

File: lang/c++/http_server/http_server.cpp [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
```
Comment (StefanJum) [inline lang/c++/http_server/http_server.cpp:32]: This is the old licence header.

**Review by StefanJum — CHANGES_REQUESTED** [review]
Works fine, see the few comments. Similar to #64, I would add the `helloworld.cpp` file and everything related to that into `lang/c++/helloworld/`

**Comment by razvand** [comment]
@StefanJum , see now.

**Review by StefanJum — APPROVED** [review]
All good
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**